docker/ccf_ci_built

```diff
@@ -11,4 +11,4 @@ FROM ${base}-${platform}
 ARG platform=sgx
 RUN mkdir /CCF
 COPY . /CCF/
-RUN mkdir /CCF/build && cd /CCF/build && cmake -GNinja -DCOMPILE_TARGET=${platform} .. && ninja
+RUN mkdir /CCF/build && cd /CCF/build && cmake -GNinja -DCOMPILE_TARGET=${platform} .. && ninja && chmod -R 777 /CCF
```

Comment:
`777` seems overly loose here?

Reply:
Yeah I have a feeling this might not even be needed anymore

Reply:
I was wrong https://github.com/microsoft/CCF/pull/4708/commits/085ed6a39292923109cf06964f94288475f2edb1

Reply:
I tried several levels of permissions and they all seem to fail so unless you think it would be harmful to 777 it I think that's okay